src/locales/en.json

```diff
@@ -169,20 +170,20 @@
     "loud_voice": "Loud talking (shouting, talking over music, singing)",
     "person": {
       "average": "An average person in your area",
-      "frontline": "An essential or front-line worker in your area",
+      "frontline": "A healthcare or social worker",
       "nonFrontline": "Someone in your area who is NOT an essential or front-line worker",
       "livingAlone": "Lives alone and only grocery shops",
-      "livingWithPartner": "Lives with partner, both people only grocery shop",
+      "livingWithPartner": "Lives with one other person, both people only grocery shop",
       "closedPod4": "In a closed pod of 4 people",
       "closedPod10": "In a closed pod of 10 people",
       "closedPod20": "In a closed pod of 20 people",
       "contact1": "Has 1 close contact (incl. roommates) who socializes but does not work. No other work or socializing.",
       "contact4": "Has 4 close contacts (incl. roommates) who socialize but do not work. No other work or socializing.",
       "contact10": "Has 10 close contacts (incl. roommates) who socialize but do not work. No other work or socializing.",
       "contactWorks": "Has 1 close contact (incl. roommates) who is an essential worker. No other work or socializing.",
-      "bars": "Goes to bars regularly",
-      "microcovid_budget_one_percent": "Uses microCOVID to maintain a risk of 1%/year (200 microcovid/week)",
-      "microcovid_budget_deci_percent": "Uses microCOVID to maintain a risk of 0.1%/year (20 microcovid/week)",
+      "bars": "Went to a bar in the last 10 days",
+      "microcovid_budget_one_percent": "Uses microCOVID to maintain a Standard Caution Budget",
+      "microcovid_budget_deci_percent": "Uses microCOVID to maintain a High Caution Budget",
```

Comment:
I wonder if this creates more confusion because people have to understand what those two budget categories mean. Which requires them to have experienced other parts of the app or documentation.

Reply:
updated. lmk what you think.

Reply:
Looks good. Thanks.